.npmignore

```diff
@@ -0,0 +1,3 @@
+browser/
+script/
```

Comment:
OK, I'm preserving it then. I couldn't fine a reference to it in the rest of the code. How is it linked?